FrontEnd/src/components/SearchPage/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
```

Comment:
Do we really need to create a new react root for the `SearchPage`?

Reply:
Thank you for feedback. Done.